Implementation of SessionContext & TestExecutionContext

anandbagmar opened the pull request:
To allow maintaining context of all test executions happening via ATD

saikrishna321 commented:
Thanks for the PR Anand. Can you please update the implementation for method parallel as well

https://github.com/AppiumTestDistribution/AppiumTestDistribution/blob/master/src/main/java/com/appium/manager/AppiumParallelMethodTestListener.java


And also can you post the usage

anandbagmar commented:
@saikrishna321 @SrinivasanTarget - 

* Updated PR for AppiumParallelMethodTestListener
* Added usage instructions here - 
https://github.com/AppiumTestDistribution/AppiumTestDistribution/wiki/SessionContext-&-TestExecutionContext

> Thanks for the PR Anand. Can you please update the implementation for method parallel as well
> 
> https://github.com/AppiumTestDistribution/AppiumTestDistribution/blob/master/src/main/java/com/appium/manager/AppiumParallelMethodTestListener.java
> 
> And also can you post the usage
